### PR TITLE
Peterbryntesson/veryfastnodes

### PR DESCRIPTION
**Pull request by peterbryntesson**
## Purpose

* Adds the possibility to have very fast nodes, nodes that are updating more frequently than once a second. Use the new --veryfastrate(vfr) setting, which sets in milliseconds the delay between updates.

## Does this introduce a breaking change?

```
[ ] Yes
[x ] No
```

## Pull Request Type
What kind of change does this Pull Request introduce?

```
[ ] Bugfix
[x ] Feature
[ ] Code style update (formatting, local variables)
[ ] Refactoring (no functional changes, no api changes)
[ ] Documentation content changes
[ ] Other... Please describe:
```

## How to Test
*  Get the code
* Run it with --vfr=100. Should update fast nodes 10 times a second

### CODE REVIEW

**Comment by peterbryntesson** [comment]
Changed PR so that it merges into oneweek instead of master.

**Comment by algattik** [comment]
Approved

**Review by HamletDRC — COMMENTED** [review]
Your commit description contains instructions to run an "npm install". I think you can safely delete that aprt.

File: src/PlcSimulation.cs [hunk 1]
```diff
@@ -88,12 +88,12 @@ public void Start()
 
             if (SlowNodeCount > 0)
             {
-                _slowNodeGenerator = TimeService.NewTimer(_plcServer.PlcNodeManager.IncreaseSlowNodes, SlowNodeRate * 1000);
+                _slowNodeGenerator = TimeService.NewTimer(_plcServer.PlcNodeManager.IncreaseSlowNodes, SlowNodeRate);
```
Comment (HamletDRC) [inline src/PlcSimulation.cs:91]: Are these two changes backwards compatible? 
Is it possible that a user is specifying a slow node rate in seconds and now we're changing the command line parameter to be milliseconds?

Comment (HamletDRC) [reply src/PlcSimulation.cs:91]: nvm, i see your change below.

**Pull request merged**